tests/test_permissions.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Permissions tests."""
+
+from __future__ import absolute_import, print_function
+
+from flask import g, url_for
+from invenio_access.models import ActionUsers
+from invenio_accounts.testutils import create_test_user
+from six import BytesIO
+
+from invenio_files_rest.models import Bucket, ObjectVersion
+from invenio_files_rest.permissions import objects_create, \
+    objects_delete_all, objects_read_all, objects_update_all
+
+
+def test_objects_permissions(app, db, dummy_location):
+    """Test permissions for files REST actions."""
+    with app.test_client() as client:
+        # Create the bucket with a single file
+        buc1 = Bucket.create()
+        key1 = "key1"  # Object key
+        stream1 = bytes('contents1')  # Contents of the data stream
+        ObjectVersion.create(buc1, key1, stream=BytesIO(stream1))
```

Comment:
Can we use ``six.BytesIO`` for stream instead of opening ``README`` files and alike ?
@lnielsen, @egabancho WDYT ?